components/ImpactCard.jsx

```diff
@@ -5,22 +5,28 @@ import { media } from "../utils";
 /*
     Example:
     - Thumbnail: image
-    - impactNum: some number
+    - isImageTop: will put the right order of image and text as specified
```

Comment:
I think this description should be better:

isImageTop - determines if the image is above the text

Current description is confusing